### PR TITLE
migrate from github.com/golang/protobuf/ptypes to google.golang.org/protobuf/types/known

### PR DESCRIPTION
**Pull request by mmorel-35**
migrate from `github.com/golang/protobuf/ptypes` to `google.golang.org/protobuf/types/known`

### Checklist prior to review

- [ ] [Link to relevant issue][1] as well as docs and UI issues -- 
- [ ] This will not break child repos: it doesn't hardcode values (.e.g "kumahq" as a image registry) and it will work on Windows, system specific functions like `syscall.Mkfifo` have equivalent implementation on the other OS --
- [ ] Tests (Unit test, E2E tests, manual test on universal and k8s) --
  - Don't forget `ci/` labels to run additional/fewer tests
- [ ] Do you need to update [`UPGRADE.md`](../blob/master/UPGRADE.md)? --
- [ ] Does it need to be backported according to the [backporting policy](../blob/master/CONTRIBUTING.md#backporting)? ([this](https://github.com/kumahq/kuma/actions/workflows/auto-backport.yaml) GH action will add "backport" label based on these [file globs](https://github.com/kumahq/kuma/blob/master/.github/workflows/auto-backport.yaml#L6), if you want to prevent it from adding the "backport" label use [no-backport-autolabel](https://github.com/kumahq/kuma/blob/master/.github/workflows/auto-backport.yaml#L8) label) --

[1]: https://github.com/kumahq/kuma/issues/3499

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -13,7 +13,7 @@ require (
 	github.com/containernetworking/cni v1.1.2
 	github.com/containernetworking/plugins v1.4.1
 	github.com/emicklei/go-restful/v3 v3.12.0
-	github.com/envoyproxy/go-control-plane v0.12.0
+	github.com/envoyproxy/go-control-plane v0.12.1-0.20240419124334-0cebb2f428b3
```
Comment (jakubdyszkiewicz) [inline go.mod:16]: @lobkovilya can we do it? I think replace won't work

**Review by jakubdyszkiewicz — COMMENTED** [review]
please don't merge it

File: go.mod (hunk 1 above)
Comment (lobkovilya) [reply go.mod:16]: yes the replace won't work in this case, I checked locally

**Comment by lahabana** [comment]
@lukidzi is this something that can be unblocked now that we moved to newer go-control-plane?

**Comment by lahabana** [comment]
@Icarus9913 this should be closed now as you've done this in another PR right?

**Comment by Icarus9913** [comment]
I think we can close it and it's finished by https://github.com/kumahq/kuma/pull/12038

**Pull request closed without merging**